src/test/java/com/diffmin/AppTest.java

```diff
@@ -35,6 +35,10 @@
 
     public static final Path DELETE_UPDATE_PATCHES = RESOURCES_BASE_DIR.resolve("delete+update");
 
+    private static final String prevPrefix = "PREV";
+
+    private static final String newPrefix = "NEW";
```

Comment:
These are constants, and thus have a special naming convention of `UPPER_CASE`. E.g. `PREV_PREFIX`.